validator/client/attest_protect_test.go

```diff
@@ -240,107 +223,6 @@ func TestAttestationHistory_BlocksDoubleAttestation(t *testing.T) {
 	}
 }
 
-func TestAttestationHistory_BlocksSurroundAttestationPostSignature(t *testing.T) {
```

Comment:
Removed because the function is no longer called concurrently now that we use a multilock on SubmitAttestation by public key